### PR TITLE
fix: license from nonexistent ICS to MIT

### PR DESCRIPTION
**Pull request by jonchurch**
~This fixes the license to be ISC which I assume it was intended to be.~

As per @wesleytodd comment, switching to MIT instead

### CODE REVIEW

**Review by wesleytodd — APPROVED** [review]

**Comment by wesleytodd** [comment]
Oh, I just thought, we should change this to mit right?

File: package.json [hunk 1]
```diff
@@ -29,7 +29,7 @@
     "type": "GitHub Sponsor",
     "url": "https://github.com/sponsors/wesleytodd"
   },
-  "license": "ICS",
+  "license": "ISC",
```
Comment (jonchurch) [inline package.json:32]:
```suggestion
  "license": "MIT",
```

Comment (jonchurch) [reply package.json:32]: @wesleytodd I'm in favor, and it will bring it in line with the license of https://github.com/pkgjs/create

**Comment by jonchurch** [comment]
I updated the LICENSE file to the MIT license text, keeping Wes as the original copyright holder bc IANAL

**Review by thescientist13 — APPROVED** [review]

**Pull request merged**